test/Serilog.Sinks.Elasticsearch.Tests/Templating/template_v7.json

```diff
@@ -71,5 +71,6 @@
                 }
             }
         }
-    }
+    },
+    "aliases": {}
```

Comment:
Can you also add a test that shows that it indeed sets the aliases when provided?

Reply:
I've added one alias in test and also changed template_v7.json.